src/components/Navbar/Navbar.js

```diff
@@ -9,6 +9,8 @@ import { ReactComponent as Friends } from '../../pics/FriendsIcon.svg';
 import { ReactComponent as Invite } from '../../pics/InviteFriends.svg';
 import { ReactComponent as Settings } from '../../pics/MySettingsIcon.svg';
 
+const VISIBILITY = { hidden: 'hidden', visible: 'visible' };
```

Comment:
The idea is to favor the use of constants instead strings, for example:
```javascript
  navbar: VISIBILITY.hidden
 // instead of
  navbar: 'hidden'
```

Reply:
I tried to create a mock api with json-server and faker for fast prototyping (It should be created in a different PR but I messed it up). This way we could create fake data with few lines of code and the behavior will be more realistic (async). To add a new collection of data we just need to modify src/server/createMockDB.js 
```javascript
// declare collections here
const friends = [];

// generate fake data inside the loop
for (let i = 0; i < times; i++) {
    // generate fake friends
    friends.push({
        id: i,
        avatar: faker.internet.avatar(),
        name: faker.name.findName()
    });
}
// Add collectiont to fake db
const data = JSON.stringify({ friends });
```
This data will be added to src/server/db.json
You can now make fetch calls like:
```javascript
  const url = 'http://localhost:3002/collection/
  const response = await fetch(url);
```
## Useful Links
+ [Faker](http://marak.github.io/faker.js/)
+ [json-server](https://github.com/typicode/json-server)